pkg/manager-fabric/manager.go

```diff
@@ -159,8 +159,13 @@ type Connection struct {
 
 var manager Fabric
```

Comment:
You could just use the manager directly. It would involve a lot more find & replace though. Your call.